sdk/onlineexperimentation/azure-resourcemanager-onlineexperimentation/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Release History
+
+## 1.0.0-beta.1 (2025-04-22)
```

Comment:
update to your planned release date

Reply:
@weidongxu-microsoft, I'm not able to push to https://github.com/azure-sdk/azure-sdk-for-java.git repo to update the PR:
```
To https://github.com/azure-sdk/azure-sdk-for-java.git
 ! [remote rejected]         smichtch/azure-resourcemanager-onlineexperimentation-4783375 -> smichtch/azure-resourcemanager-onlineexperimentation-4783375 (permission denied)
error: failed to push some refs to 'https://github.com/azure-sdk/azure-sdk-for-java.git'
```
I pushed the changes here (public repo in Azure org): https://github.com/Azure/azure-sdk-for-java/tree/smichtch/azure-resourcemanager-onlineexperimentation-4783375 if you could pull them into this PR.
And updated release date and the TypeSpec commit to include https://github.com/Azure/azure-rest-api-specs/pull/34291 + re-ran `tsp-client update`

Reply:
I've pulled the last 2 commit from your PR.

Let us know what you are ready to merge, by either approve the PR, or comment as such.